lib/animina_web/potential_partner.ex

```diff
@@ -64,12 +64,12 @@ defmodule AniminaWeb.PotentialPartner do
   end
 
   defp partner_green_flags_query(query, user) do
-    green_flags =
-      get_user_flags(user.id, :green)
+    white_flags =
```

Comment:
I am not happy with the function name. Even after reading the code I am not 100% if I understand what this function does.